concent_api/utils/helpers.py

```diff
@@ -2,14 +2,27 @@
 import base64
 import binascii
 import datetime
+import os
 import time
 
+from django.conf                    import settings
+from django.core.checks             import Error
 from django.utils                   import timezone
 
 from golem_messages                 import message
 from golem_messages.datastructures  import FrozenDict
 
 
+def storage_cluster_certificate_path_check(app_configs = None, **kwargs):
+    certificate_path = settings.STORAGE_CLUSTER_SSL_CERTIFICATE_PATH
+    if certificate_path != '':
+        if not os.path.exists(certificate_path):
+            return [Error("File not found")]
+        if os.path.splitext(certificate_path)[1] != '.crt':
+            return [Error(f"{certificate_path} is not a SSL certificate")]
```

Comment:
@cameel - but the file is going to be PEM format, right? There is an option to try to decode cert and if it fails, add an error. Eg.
```
from cryptography import x509
from cryptography.hazmat.backends import default_backend

backend = default_backend()
with open('./cert.crt', 'rb') as f:
    crt_data = f.read()
    cert = x509.load_pem_x509_certificate(crt_data, backend)
```
Should this suffice?

Reply:
Yeah, I think that we can safely assume that it will be the PEM format.

This check would be great. You could wrap it into a function and put in `utils`. Just give it a name that makes it clear that it only validates the format and not whether the certificate can be trusted.

But I don't want to introduce a dependency on `cryptography` just for this. Golem considered using it instead of `pyelliptic` but eventually decided not to because not everything was there. golem-messages already pulls a ton on cryptographic packages (here's Golem's dependency list: https://github.com/golemfactory/golem/blob/develop/requirements.txt). I think `pyelliptic` should be able to do this because it uses OpenSSL which can work with certificates.

But don't get too deep into this. If this requires extra work or research it's better to just skip the check for now and create a separate issue for it.

Reply:
isn't `pyelliptic` deprecated?
https://github.com/yann2192/pyelliptic/issues/50#issuecomment-297018932

Reply:
It is. But Golem is using it and they have their own fork with OpenSSL 1.1. Compatibilty.

Reply:
ok, done with simple `isfile` check